wallet/hardware.md

```diff
@@ -25,10 +25,11 @@ This type of set-up does come with additional hurdles that may be more difficult
 
 1.  [BitBox02](#bitbox02)
 2.  [Coldcard](#coldcard)
-3.  [Passport](#passport)
-4.  [Seed Signer](#seed-signer)
-5.  [Specter DIY](#specter-diy)
-6.  [Trezor](#trezor)
+3.  [Jade](#Jade)
```

Comment:
needs to be lower case or the link won't work

Reply:
done with [e9ac1b3](https://github.com/BitcoinQnA/Bitcoiner.Guide/pull/39/commits/e9ac1b39209d9705dd6aa0935799409c910798ea)